css/style.css

```diff
@@ -15,5 +18,247 @@ body {
+  border-radius: 10px;
+  
+  
+}
+
+.three_imgs{
+  
+  height: 300px;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  margin:auto;
+  width: 90%;
+  margin-bottom: 200px;
+  
+}
+
+.img_1{
+  height: 200px;
+  width: 200px;
+  margin: auto;
+  
+}
+
+.Every_needs{
+  text-align: center;
+  margin-top: 100px;
+}
+
+
+  .header{
+    display: flex;
+    background-color: white;  
+    height: 40px;
+    width: 83%;
+    margin: auto;
+    margin-top: 5px;
+    margin-bottom: 40px;
+    padding: 20px;
+    justify-content: space-between;
+    
+  }
+
+  .nav_ul{
+    list-style: none;
+    margin: 0;
+    padding: 0;
+    overflow: hidden;
+    
+    
+  }
+
+
+
+  /* ######################################################b2 */
+
+  
+  a {
+    color: rgb(1, 1, 1);
+    text-align: center;
+    padding: 14px 16px;
+    text-decoration: none;
+    flex-direction: row;
+    font-size: 200%;
+  }
+
+  .ad_back_img{
+    
+    width: auto;
+    /* border-radius: 5px; */
+    text-align: center;
+    padding: 20px;
+    margin-bottom: 200px;
+
+    height: auto;
+    border: 4px solid yellow;
+    
+  }
+
+  h3{
+
+    margin-top: 20px;
+    font-size: 60px;
+    font-family: 'Catamaran', sans-serif;
+  }
+
+  h4{
+    font-size: 50px;
+    font-family: 'Catamaran', sans-serif;
+  }
+
+  .main_back_img {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 83%;
+    height: 800px;
+    margin: auto;
+    margin-top: 5px;
+    background-image: url("../first-background.jpg");
+    background-size: 100% 100%;
+    background-repeat: no-repeat; 
+    border: 2px solid rgb(49, 199, 49); 
+  }
+
+
+
+  .main_nav{
+    width: 80%;
+    margin: auto;
+  }
+
+
+  .back_img{
+    width: 1800px;
+    height: 750px;
+    margin: auto;
+    padding: auto;
+    display: flex;
+    margin-top: 10px;
+  }
+  
+  .fav_ico{
+    display: flex;
+    height: 35px;
+    width: 35px;
+    max-height: 90px;
+    width: auto;
+    margin-left: 0;
+    padding: 5px;
+    justify-content: start;
+    
+  }
+
+  button{
+    background-color: rgb(228, 92, 19);
+    color: #fff;
+    border-radius: 17px;
+    font-size: 50px;
+    /* padding: 30px;
+    width: 500px; */
+    border-color: rgb(228, 92, 19);
+    }
+
+  .bget_garma_1{
+  background-color: rgb(228, 92, 19);
+  color: #fff;
+  border-radius: 17px;
+  font-size: 50px;
+  padding: 30px;
+  width: 500px;
+  border-color: rgb(228, 92, 19);
+  }
+
+  hr{
+    margin-top: 150px;
+  }
+
+  .join_us{
+    text-align: center;
+  }
+
+  .footer_imgs{
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    height: 100px;
+  }
+
+  .footer_ico{
+    border:3px solid rgb(218, 219, 218);
+    display: flex;
+    width: 50px;
+    height: 50px;
+    margin-top: 10px;
+    border-radius: 70px;
+    padding: 15px;
+    
+    
+    
+  }
+  .contener{
+    display: grid;
+    
+    grid-template-areas: 
+    "b_img b_img b_text b_text b_text"
```

Comment:
Repeating the same grid area name is a cool way to keep named areas but also be able to specify different widths for the columns - nice one :muscle: 

The only improvement I would suggest is that as you only have one row, you don't need to repeat the line, i.e. this would work just as well:

```css
grid-template-areas:  b_img b_img b_text b_text b_text;
```

Here's a screenshot demonstrating:

![image](https://user-images.githubusercontent.com/1737376/183218879-24f42791-81e2-4e75-9ef4-aeb177e86624.png)

I find these options in the dev tools really helpful to visualise what's going on when using CSS grid:

![image](https://user-images.githubusercontent.com/1737376/183218984-6997cade-0f58-4ec1-8ec0-cf19d5c4a196.png)